### ISSUE TITLE
Position of Popup

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-new-window-warnings#7]
A site was being tested by a user with very poor vision so using a lot of font size increase and magnification. One issue was that the popup was not visible or obscured some of the link words (the link was about 10 words long and ran into multiple lines).

This also occurs when the link is that the bottom of a screen. If a link is at the bottom of what is currently visible, the popup is below and offscreen.

It was also reported that since the popup is attached to the mouse movement, users with physical issues such as shaking hands would see a constantly moving message and not be able to read it easily.

The request from the testers was that the popup alert be positioned above the element, and attached to the “top left” corner of the <a> element. So that it would not move on mouse movement, and also so it would be visible for content lower on the screen.

I’m sure there is some room for debate about being attached to “top” of element vs “bottom” of element might be better. But the testers felt that attaching to top would be better as they are more likely to be on a link at the bottom of the screen than at the top.

Very much appreciate the plugin and keep up the good work Equalize Digital!

Support Ticket: https://wordpress.org/support/topic/change-position-of-popup/

**Comment by joedolson** [comment]
I suggest observing whether one side of the element is currently outside of the viewport, and choosing the side that's in the viewport when possible. (Considering the hypothetical where neither side is in the viewport.)
